src/components/Menu/OptionalMenu.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Paper from 'material-ui/Paper';
+import MenuHeader from './MenuHeader';
+import ImageGallery from './ImageGallery';
+import './OptionalMenu.css';
+
+class OptionalMenu extends React.Component {
+    closeMenu = () => {
+        document.getElementsByClassName("menu")[0].style.display = "none";
```

Comment:
You can create `Menu` and `Board` components, and there are a lot of ways to communicate between components, so you can avoid these direct assignments. Applies to other `document.get*` calls too, I think you can try to get rid of all of them